gradle.properties

```diff
@@ -1,8 +1,5 @@
 android.useAndroidX=true
 android.enableJetifier=true
-android.debug.obsoleteApi=true
 org.gradle.jvmargs=-Xmx4g
-org.gradle.configuration-cache=true
-org.gradle.caching=true
```

Comment:
This feels a bit sad. Are MR not working with Gradle cache?

Reply:
Oh, yeah, that was an attempt to fix a build issue. I will try to rollback this change :)